waffle-mock-contract/test/proxiedTest.ts

```diff
@@ -57,5 +57,21 @@ export function mockContractProxiedTest(provider: MockProvider) {
       await mockContract.mock.read.returns(1);
       expect(await proxy.readCapped()).to.eq(1);
     });
+
+    it('calledOnContract with mock contract', async () => {
```

Comment:
What's the reason for adding these tests?

Reply:
I wrote these for #809 and forgot about it.